authors/apps/authentication/test_user_registration.py

```diff
@@ -0,0 +1,117 @@
+import json
+from django.urls import reverse
+from rest_framework.views import status
+from rest_framework.test import APITestCase,APIClient
+
+class RegistrationTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.login_url =reverse('authentication:auth-login')
+        self.signup_url = reverse('authentication:auth-register')
+        self.signup_data = {
+  "user":{
```

Comment:
You have done good work, but I would recommend that you indent your code according to pep8 standards. You can check more on pep8 standards here https://www.python.org/dev/peps/pep-0008/#indentation.